build-system/SERVING.md

```diff
@@ -52,6 +52,7 @@ rm /path/to/cdn/production/alp.js.bak
 # make sure and prepend the global production config to main binaries
 gulp prepend-global --target /path/to/cdn/production/v0.js --prod
 gulp prepend-global --target /path/to/cdn/production/alp.js --prod
+gulp prepend-global --target /path/to/3p/cdn/production/f.js --prod
```

Comment:
@cramforce are we ok with recommending this? we dont need to do it for our f.js but it definitely needs to be done for other caches